hazelcast/src/main/java/com/hazelcast/spi/impl/eventservice/impl/EventServiceImpl.java

```diff
@@ -583,7 +583,8 @@ private void sendEvent(Address subscriber, EventEnvelope eventEnvelope, int orde
             }
         } else {
             Packet packet = new Packet(serializationService.toBytes(eventEnvelope), orderKey)
-                    .setPacketType(Packet.Type.EVENT);
+                    .setPacketType(Packet.Type.EVENT)
+                    .raiseFlags(Packet.FLAG_4_0);
```

Comment:
minor: could move raising `FLAG_4_0` to `Packet` constructor as done in other PR

Reply:
Sure, I was planning on doing it 👍

Reply:
Done, updated in this and the 3.12.z PR. Now I only had to unset the flag when sending to 3.12 members and set the FLAG_3_12 flag, which is kinda ugly but the rest of the code using Packets is now unaffected.